.github/workflows/pr_author_auto_assign.yaml

```
@@ -0,0 +1,21 @@
+name: PR author auto assign
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)